samples/CSharp/SourceGenerators/SourceGeneratorSamples/MustacheGenerator.cs

```diff
@@ -19,7 +20,7 @@ public void Execute(SourceGeneratorContext context)
         {
             string attributeSource = @"
     [System.AttributeUsage(System.AttributeTargets.Assembly, AllowMultiple=true)]
-    public class MustacheAttribute: System.Attribute
+    internal class MustacheAttribute: System.Attribute
```

Comment:
```suggestion
    #nullable enable
    internal sealed class MustacheAttribute: System.Attribute
```

Reply:
Reverted nullable change in ac5b4937.

Reply:
This change request is still relevant.

Reply:
Nullable reverted, Attribute made 'internal sealed'. If there is anything else, please reactivate.